pkg/pb/orders.proto

```diff
@@ -64,15 +65,14 @@ message PieceHash {
     bytes piece_id = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false];
     // hash of the piece that was/is uploaded
     bytes hash = 2;
-    // signature either satellite or storage node
-    bytes signature = 3;
     // size of uploaded piece
-    int64 piece_size = 4;
-    // timestamp when upload occur
-    google.protobuf.Timestamp timestamp = 5 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+    int64 piece_size = 3;
+    // timestamp when upload occurred
+    google.protobuf.Timestamp timestamp = 4 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+    // signature either satellite or storage node
+    bytes signature = 5;
```

Comment:
Why make these changes?

Reply:
Just to make it look pretty, and these ones we still can change.